### ISSUE TITLE
Two questions about paper

### ISSUE THREAD

**Issue by wjczf123** [deeplearning-wisc/multi-label-ood#1]
Very nice work! I have two questions about this paper:

1. Does the validation set have OOD data or only IND data?
2. LOF fits on the validation set instead of training set. Why? Does this affect the results? 

Thanks!
